### DISCUSSION TITLE
How to stitch point clouds with no overlap?

### DISCUSSION THREAD

**Discussion by amal-k-ajay** [isl-org/Open3D#6995] [Q&A]
Hi guys, I'm looking for ways to merge two point clouds from two tof cameras kept opposite to each other by using any algorithms or techniques. I tried using ICP but since there is no overlap between the point clouds it's not working. I also tried to manually transform the point clouds and merge them using know orientation and position but it's not accurate. Any help is appreciated. Thanks in advance!

**Comment by prabhat-twr** [comment]
I am also facing same issue.

**Comment by amal-k-ajay** [comment]
Hi @astro-wizard ,
I solved my problem. If the sensors are going to be stationary, then you can perform point-set registration. Pick a few point sets from both clouds to calculate the spatial transformation and merge them.
